ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/AttestationUtil.java

```diff
@@ -99,24 +99,62 @@
   }
 
   public static List<PendingAttestation> get_previous_epoch_justified_attestations(
-      BeaconState state) {
-    // todo
-    return new ArrayList<PendingAttestation>();
+      BeaconState state) throws Exception {
+    // Get previous and current epoch
+    UnsignedLong current_epoch = BeaconStateUtil.get_current_epoch(state);
+    UnsignedLong previous_epoch = BeaconStateUtil.get_previous_epoch(state);
+
+    // Get previous and current_epoch_attestations
+    List<PendingAttestation> attestations =
+        AttestationUtil.get_epoch_attestations(state, previous_epoch);
+
```

Comment:
You don't need to prefix the call to `get_epoch_attestations` with `AttestationUtil` since this code is in `AttestationUtil`.